lib/pure/collections/sharedlist.nim

```diff
@@ -92,4 +92,10 @@ proc deinitSharedList*[A](t: var SharedList[A]) =
   clear(t)
   deinitLock t.lock
 
+proc initSharedList*[A](): SharedList[A] {.deprecated.} =
+  # This is not posix compliant, may introduce undefined behavior. Use `init` instead.
```

Comment:
'##' instead of '#' for documentation comments.

Reply:
Are you sure you want to make deprecated stuff show up in the docs? Because I've intentionally used '#' instead of '##'.

Reply:
Yes I am sure as long as ``{.deprecated: "Use 'init' instead".}`` does not work.